src/components/Mapathons/FiltersDialog.js

```diff
@@ -0,0 +1,298 @@
+            onClickHandler={this.props.hide}
+            style={{ padding: '0rem' }}
+          >
+            <Icon
+              glyph="cross"
+              size={1}
+              backgroundColor={colors.backgroundColor}
+              disabled={this.props.sendingRequest}
+              onClickHandler={this.props.hide}
+              color={colors.darkestGrey}
+            />
+          </Button>
+          <Title>
+            {this.context.intl.formatMessage(messages.filtersTitle)}
+          </Title>
+        </Header>
+
+        <Content>
+          <ButtonGroupWrapper style={{ marginBottom: '1.5rem' }}>
+            <Label>
+              Number of Reviews
+            </Label>
+            <ButtonGroup size="sm">
+              <Button
+                disabled={this.props.sendingRequest}
+                onClick={() => this.updateNumOfReviews(1)}
+                className={`${
+                  this.state.numberOfReviews === 1
+                    ? 'btn-secondary is-active'
+                    : 'btn-secondary'
+                }`}
+              >
+                Ascending
+              </Button>
+              <Button
+                disabled={this.props.sendingRequest}
+                onClick={() => this.updateNumOfReviews(-1)}
+                className={`${
+                  this.state.numberOfReviews === -1
+                    ? 'btn-secondary is-active'
+                    : 'btn-secondary'
+                }`}
+              >
+                Descending
+              </Button>
+            </ButtonGroup>
+          </ButtonGroupWrapper>
+          <ButtonGroupWrapper style={{ marginBottom: '1.5rem' }}>
+            <Label>
+              Date
+            </Label>
+            <ButtonGroup size="sm">
+              <Button
+                disabled={this.props.sendingRequest}
+                onClick={() => this.updateDate(1)}
+                className={`${
+                  this.state.date === 1
+                    ? 'btn-secondary is-active'
+                    : 'btn-secondary'
+                }`}
+              >
+                Ascending
+              </Button>
+              <Button
+                disabled={this.props.sendingRequest}
+                onClick={() => this.updateDate(-1)}
+                className={`${
+                  this.state.date === -1
+                    ? 'btn-secondary is-active'
+                    : 'btn-secondary'
+                }`}
+              >
+                Descending
+              </Button>
+            </ButtonGroup>
+          </ButtonGroupWrapper>
+          <Label>
+            Location
+          </Label>
+          <SelectBox
+            id="radius"
+            value={this.state.geolocation.radius}
+            options={options}
+            style={{ marginBottom: '1.5rem' }}
+            handleValueChange={this.updateGeolocation}
+            ariaLabel="Filter by Type"
+          />
+
+          <CustomButtonGroup
```

Comment:
If this is enabled and I click on it again, it doesn't disable the selection (like for the venue filter buttons)